rclcpp/include/rclcpp/init_options.hpp

```diff
@@ -42,6 +42,16 @@ class InitOptions
   RCLCPP_PUBLIC
   InitOptions(const InitOptions & other);
 
+  /// Return `true` if logging should be initialized.
```

Comment:
@ivanpauno initialized when? I presume when the context this options apply to is. Mind to be explicit?

Reply:
See https://github.com/ros2/rclcpp/pull/998/commits/0c95f168d584923b8ccb00d6080561df82b95db5